libs/service-portal/settings/src/screens/DelegationGreeting/DelegationGreeting.tsx

```diff
@@ -28,7 +27,7 @@ export const DelegationGreeting: ServicePortalModuleComponent = () => {
               defaultMessage: 'Mín réttindi',
             })}
           </Text>
-          <Tag variant="mint">
+          <Tag variant="blue">
```

Comment:
I think mint is correct here

Reply:
According to the design (Settings/NavigationScreen) the variant of "Í vinnslu" tag changed from "mint" to "blue". I was under the impression that the variant for the same tag should also be updated on the DelegationGretting screen.